src/main/java/com/kainos/ea/dao/RolesDao.java

```diff
@@ -15,14 +15,20 @@ public RolesDao(){
     public List<JobRole> getAllRoles(Connection c) throws SQLException {
         try {
             Statement st = c.createStatement();
-            ResultSet rs = st.executeQuery(
-                    "SELECT id, kainos_job_title "
-                            + "FROM job_role;");
+            ResultSet rs = st.executeQuery("SELECT job_role.id, job_role.kainos_job_title, capability.capability_name\n" +
+                    "FROM job_role JOIN job_family\n" +
+                    "ON (job_role.job_family_id = job_family.id)\n" +
+                    "JOIN capability " +
+                    "ON (job_family.capability_id = capability.id);");
 
             List<JobRole> jobRoles = new ArrayList<>();
 
             while (rs.next()) {
-                jobRoles.add(new JobRole(rs.getInt("id"), rs.getString("kainos_job_title")));
+                JobRole role = new JobRole(
+                        rs.getInt("id"),
+                        rs.getString("Kainos_Job_Title"),
```

Comment:
looks like some inconsistencies in column names

Reply:
names fixed